random1on1/api/config.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+
+class Random1on1BotConfig(object):
+    # TODO: Add configuration values and precondition checks here
+
+    def __init__(
+        self,
+        random1on1_role: str,
+        history_channel: str,
+        logging_channel: str,
+        announcement_channel: str,
+        dm_matches: bool,
+        announce_matches: bool,
+    ):
+        self.random1on1_role = random1on1_role
+        self.history_channel = history_channel
+        self.logging_channel = logging_channel
+        self.announcement_channel = announcement_channel
+        self.dm_matches = dm_matches
+        self.announce_matches = announce_matches
+
+    def fromJSON(json_data: str) -> Random1on1BotConfig:
+        return Random1on1BotConfig(**json.loads(json_data))
+
+    def toJSON(self) -> str:
+        return json.dumps(
+            self, default=lambda obj: obj.__dict__, sort_keys=True, indent=4
+        )
```

Comment:
I would prefer if we stick to a style for method and function naming. The most common is `lower_with_under`.

Also, consider renaming the methods to `get_data` and `save_data`. The format of the data in disk should be somewhat independent of it's functionality.

Reply:
I'll change the name styles for sure but have a slight preference for saying json because it's returning a json string & can only accept json strings. 

It's not doing any writing to disc or anything in this method